configs/triton_services/aws.yaml

```diff
@@ -0,0 +1,6 @@
+head_node:
+  instance_type: g5.4xlarge
+  resources:
+    cpu: 0
+worker_nodes: []
+auto_select_worker_config: true
```

Comment:
Why are you using a GPU head node? This is an anti-pattern.

Reply:
Triton requires model to be built on a GPU instance

Reply:
Ok as long as this template is not on the home page of Anyscale-hosted OA, I don't have an opinion -- but if it does show up there, then we should try harder to make this a CPU head node so that startup time is very fast, and people aren't paying for idle gpu cost while they are writing code

In either case, GPU head node is anti-pattern

Reply:
Can you put the model compilation inside a ray task?

Reply:
Yes, I'm going to try that tonight or sometime tomorrow